recipes/geant4/run_test.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-cmake ${PREFIX}/share/Geant4-${PKG_VERSION}/examples/basic/B1
```

Comment:
I don't think this will fix it, do you mind if I push a patch to this branch later today?

Reply:
Sure, but please wait until this build has finished, just in case it does fix the problem on macOS :smile:

Reply:
So it failed again... not sure why but it feel suspicious to me that the build and test-phase do not appear to use the same packages (default vs conda-forge).

```
The following NEW packages will be INSTALLED:

    bzip2:           1.0.6-h1de35cc_1002        conda-forge
    ca-certificates: 2019.3.9-hecc5488_0        conda-forge
    cctools:         895-h7512d6f_0             conda-forge
    clang:           4.0.1-h662ec87_0           conda-forge
    clang_osx-64:    4.0.1-h1ce6c1d_11          defaults   
    clangxx:         4.0.1-hc9b4283_0           defaults   
    clangxx_osx-64:  4.0.1-h22b1bf0_11          defaults   
    cmake:           3.14.0-hdd2e4aa_0          conda-forge
    compiler-rt:     4.0.1-h5487866_0           conda-forge
    curl:            7.64.1-h22ea746_0          conda-forge
    expat:           2.2.5-h0a44026_1002        conda-forge
    krb5:            1.16.3-hcfa6398_1001       conda-forge
    ld64:            274.2-h7c2db76_0           conda-forge
    libcurl:         7.64.1-h16faf7d_0          conda-forge
    libcxx:          4.0.1-h579ed51_0           defaults   
    libcxxabi:       4.0.1-hebd6815_0           defaults   
    libedit:         3.1.20170329-hcfe32e1_1001 conda-forge
    libssh2:         1.8.2-hcdc9a53_2           conda-forge
    libuv:           1.26.0-h1de35cc_0          conda-forge
    llvm:            4.0.1-hc748206_0           conda-forge
    llvm-lto-tapi:   4.0.1-h6701bc3_0           conda-forge
    make:            4.2.1-h1de35cc_2004        conda-forge
    ncurses:         6.1-h0a44026_1002          conda-forge
    openssl:         1.1.1b-h01d97ff_2          conda-forge
    rhash:           1.3.6-h1de35cc_1001        conda-forge
    tk:              8.6.9-ha441bb4_1001        conda-forge
    xz:              5.2.4-h1de35cc_1001        conda-forge
    zlib:            1.2.11-h1de35cc_1004       conda-forge
```

```
The following NEW packages will be INSTALLED:

    bzip2:           1.0.6-h1de35cc_1002        conda-forge
    ca-certificates: 2019.3.9-hecc5488_0        conda-forge
    cctools:         895-1                      defaults   
    clang:           4.0.1-1                    defaults   
    clang_osx-64:    4.0.1-h1ce6c1d_11          defaults   
    clangxx:         4.0.1-1                    defaults   
    clangxx_osx-64:  4.0.1-h22b1bf0_11          defaults   
    cmake:           3.14.0-hdd2e4aa_0          conda-forge
    compiler-rt:     4.0.1-hcfea43d_1           defaults   
    curl:            7.64.1-h22ea746_0          conda-forge
    expat:           2.2.5-h0a44026_1002        conda-forge
    geant4:          10.5.0-h0437068_0          local      
    icu:             58.2-h0a44026_1000         conda-forge
    krb5:            1.16.3-hcfa6398_1001       conda-forge
    ld64:            274.2-1                    defaults   
    libcurl:         7.64.1-h16faf7d_0          conda-forge
    libcxx:          4.0.1-hcfea43d_1           defaults   
    libcxxabi:       4.0.1-hcfea43d_1           defaults   
    libedit:         3.1.20170329-hcfe32e1_1001 conda-forge
    libssh2:         1.8.2-hcdc9a53_2           conda-forge
    libuv:           1.26.0-h1de35cc_0          conda-forge
    llvm:            4.0.1-1                    defaults   
    llvm-lto-tapi:   4.0.1-1                    defaults   
    make:            4.2.1-h1de35cc_2004        conda-forge
    ncurses:         6.1-h0a44026_1002          conda-forge
    openssl:         1.1.1b-h01d97ff_2          conda-forge
    rhash:           1.3.6-h1de35cc_1001        conda-forge
    tk:              8.6.9-ha441bb4_1001        conda-forge
    xerces-c:        3.2.2-h44e365a_1001        conda-forge
    xz:              5.2.4-h1de35cc_1001        conda-forge
    zlib:            1.2.11-h1de35cc_1004       conda-forge
```

Reply:
That's fine, conda-forge's compiler packages come from anaconda to avoid issues with building the compilers and to improve the interoperability between the channels. Some of them have been manually copied over but these haven't.